### ISSUE TITLE
Non Zero Exit Code! -1073741819

### ISSUE THREAD

**Issue by DigDug5** [Aloshi/EmulationStation#426]
Anyone have any idea what this might mean? Im using retroarch here is my config file

<system>
		<name>gba</name>
		<fullname>Game Boy Advance</fullname>
		<path>D:\Games\GB, GBC, GBA\Games\Gameboy Advanced</path>
		<extension>.gba</extension>
		<command>D:\Games\retroarch\retroarch.exe D:\Games\retroarch\libretro\vbam_libretro.dll    "%ROM_RAW%"</command>
		<platform>gba</platform>
		<theme>gba</theme>
	</system>

Any help would be greatly appreciated! Thanks!

**Comment by Aloshi** [comment]
This is almost certainly from RetroArch's side (probably from the vbam core to be precise).

Try running
```D:\Games\retroarch\retroarch.exe D:\Games\retroarch\libretro\vbam_libretro.dll "D:\Games\GB, GBC, GBA\Games\Gameboy Advanced\Final Fantasy Tactics Advance.gba"```
from the command prompt manually and see if it works. You might get more detailed error messages this way.

**Comment by DigDug5** [comment]
Yeah i think you are correct. I am very new to this but are you saying to just launch that via cmd or is there a place in retroarch's folder that i can launch a command prompt specifically for that

**Comment by Aloshi** [comment]
Just launch it from cmd.

**Comment by DigDug5** [comment]
Retroarch: [libretro INFO] :: Frontend supports RGB565 - will use that instead of XRGB1555
Retroarch: [libretro INFO] :: GameID in Rom is : V
Retroarch: [libretro INFO] :: RTC = 0
Retroarch: [libretro INFO] :: flashSize = 65536
Retroarch: [libretro INFO] :: cpuSaveType = 0
Retroarch: [libretro INFO] :: mirroringEnable=0

And here is what Retroarch says under problem details. 

Problem signature:
  Problem Event Name:	APPCRASH
  Application Name:	retroarch.exe
  Application Version:	0.0.0.0
  Application Timestamp:	00000000
  Fault Module Name:	vba_next_libretro.dll
  Fault Module Version:	0.0.0.0
  Fault Module Timestamp:	532b9c14
  Exception Code:	c0000005
  Exception Offset:	000000000005145f
  OS Version:	6.3.9600.2.0.0.768.98
  Locale ID:	1033
  Additional Information 1:	5e9c
  Additional Information 2:	5e9c9d074c327df302169327cdb7d949
  Additional Information 3:	3681
  Additional Information 4:	368147d8f4dd263177e13e593f6f1c42
